modelforge/train/training.py

```diff
@@ -45,6 +45,10 @@
 from modelforge.train.losses import LossFactory, create_error_metrics
 from modelforge.train.parameters import RuntimeParameters, TrainingParameters
 
+import matplotlib
+
+matplotlib.use("Agg")
```

Comment:
Need a comment here saying this is to get rid of tkinter errors in profilers by using background mode of matplotlib without calling GUI functions.